src/ServiceControl.Monitoring/Timings/IntervalsStore.cs

```diff
@@ -3,33 +3,35 @@
     using System;
     using System.Collections.Concurrent;
     using System.Collections.Generic;
+    using System.Linq;
     using System.Threading;
+    using Infrastructure;
     using Metrics.Raw;
 
-    public abstract class TimingsStore
+    public abstract class IntervalsStore : IKnowAboutEndpoints
```

Comment:
I'm not a native English speaker but this name is somewhat strange `IntervalsStore`. Shouldn't it be `IntervalStore`?